snuba/query/processors/granularity_processor.py

```diff
@@ -39,3 +41,53 @@ def process_query(self, query: Query, query_settings: QuerySettings) -> None:
                 Literal(None, granularity),
             )
         )
+
+
+PERFORMANCE_GRANULARITIES = [(60, 1), (3600, 2), (86400, 3)]
+DEFAULT_MAPPED_GRANULARITY_ENUM = 1
+
+
+class MappedGranularityProcessor(QueryProcessor):
+    """
+    Use the granularity set on the query to filter on the granularity column,
+    supporting generic-metrics style enum mapping
+    """
+
+    def __init__(
+        self,
+        accepted_granularities: Sequence[Tuple[int, int]],
```

Comment:
Why do you pass these in? Everywhere you are using this class it is using the constants above.